webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -1,6 +1,7 @@
 import { Tabs } from 'antd';
 import React, { useCallback, useEffect, useState } from 'react';
-import { useHistory, useParams } from 'react-router';
+import { useParams } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
```

Comment:
**nit:** one line, maybe IDE needs adjustment to do this?

Reply:
I forgot to combine them when I replace `react-router-dom-v5-compat` with `react-router-dom`. Nice catch!